dpcpp/components/cooperative_groups.dp.hpp

```diff
@@ -353,8 +353,7 @@ template <unsigned Size, typename Group>
 __dpct_inline__
     std::enable_if_t<(Size > 1) && Size <= 64 && !(Size & (Size - 1)),
                      detail::thread_block_tile<Size>>
-        tiled_partition
-    [[intel::reqd_sub_group_size(Size)]] (const Group& group)
```

Comment:
we should keep this.
 we can use old compiler with sycl1.2.1 to check whether the subgroup size is changed or not.
 It's helpful when porting the block Jacobi and threshold selection. Otherwise, SYCL will just ignore the setting without any compile error

Reply:
Because dpcpp 2022 also remove the propagation even if it is on old standard, I withdraw the comment here